javascript/javascript3/week1/giphyapi/gif.js

```diff
@@ -0,0 +1,28 @@
+let errorMessage = document.getElementById('content');
+
+function fetchGif() {
+
+    errorMessage.innerText = '';
+
+    let searchValue = document.getElementById('search').value;
```

Comment:
why not `const`?

Reply:
Thank you for the review rasmus. I will use const and avoid let hereafter.